src/read.rs

```diff
@@ -20,27 +20,21 @@ impl<'a, I> ReadHtml<'a, I> where I: Iterator<Item=Event<'a>> {
 }
 
 impl<'a, I> io::Read for ReadHtml<'a, I> where I: Iterator<Item=Event<'a>> {
-    fn read(&mut self, mut buf: &mut [u8]) -> io::Result<usize> {
-        while self.current.len() < buf.len() {
```

Comment:
Does this loop give us any real performance gain? As long as no events serialize to empty string, this is ok...